keyboards/ambergon/multipleglove/keyboard.json

```
@@ -0,0 +1,59 @@
+{
+    "manufacturer": "ATSUTO_NAKAYAMA",
+    "keyboard_name": "MultipleGlove",
+    "maintainer": "ambergon",
+    "bootloader": "atmel-dfu",
```

Per the [github project](https://github.com/ambergon/MultipeGlove/tree/master) and the microcontroller you used, this would be the most appropriate configuration. If you do want to retain the `atmel-dfu` bootloader setting, use `elite_c` instead here.
```suggestion
    "development_board": "promicro",
```